### ISSUE TITLE
Tool Presets Preferences error

### ISSUE THREAD

**Issue by 35743** [Pullusb/storytools#66]
Error in the Add-on Preferences > Tool Presets tab.

![image](https://github.com/user-attachments/assets/6bae9c44-c8a5-4132-8360-4b20d71eed68)
The console output:
```
AttributeError: 'NoneType' object has no attribute 'keymap_items'
Traceback (most recent call last):
  File "C:\Program Files\Blender Foundation\Blender 4.3\4.3\scripts\startup\bl_ui\space_userpref.py", line 2424, in draw_addon_preferences
    draw(context)
  File "C:\Users\User\AppData\Roaming\Blender Foundation\Blender\4.3\scripts\addons\storytools-master\preferences.py", line 521, in draw
    for kmi in reversed(km.keymap_items):
                        ^^^^^^^^^^^^^^^
AttributeError: 'NoneType' object has no attribute 'keymap_items'
```

blender 4.3.0, Windows 11

**Comment by Pullusb** [comment]
Thank you for the report,

Will fix tomorrow.

Forgot to mention on the readme that the port to gpv3 (#58) is not 100% complete yet.
But will be close once this issue is fixed.

**Comment by Pullusb** [comment]
Fixed in 6322923b23e3dc75488bdd895cf7f446a58b29a9
and improved in 1df96b3444e36c21e0b48a9cffd5dcbcee925025 

Now expose ordering and order is correctly replicated on toolpresets gizmos (on reload tool presets button is used).
(note: Could have used a _UI list_ for a more user friendly orderring, maybe later, low priority)
